src/toil/wdl/wdl_functions.py

```diff
@@ -839,6 +839,19 @@ def write_map(in_map: Dict[str, str],
     return path
 
 
+def wdl_range(num: int) -> List[int]:
+    """
+    Given an integer argument, the range function creates an array of integers of
+    length equal to the given argument.
+
+    WDL syntax: Array[Int] range(Int)
+    """
+    assert isinstance(num, int) and num >= 0, \
+        f'range() requires an integer greater than or equal to 0 (but got {num})'
```

Comment:
Nice!  Just tested this and I see you've implemented the exact same message that Cromwell gives.  Great attention to detail.